### ISSUE TITLE
scss/operator-no-unspaced: Interprets minus-sign of negative value as operator

### ISSUE THREAD

**Issue by jens-duttke** [kristerkari/stylelint-scss#450]
If a function is wrapped into `#{` and `}`, the minus-sign of negative numbers in the arguments is interpreted as operator.

```scss
div {
  // This throws `Expected single space after "-" (scss/operator-no-unspaced)`:
  --my-var: #{scale-color(#fff, $lightness: -75%)};

  // This works fine:
  color: scale-color(#fff, $lightness: -75%);
}
```

I'm using:
```
"stylelint": "13.0.0",
"stylelint-scss": "3.14.2",
```

**Comment by kristerkari** [comment]
Thanks!

**Comment by kristerkari** [comment]
Fix released in v3.16.1
https://github.com/kristerkari/stylelint-scss/releases/tag/3.16.1
